jobs/ci-kubernetes-cross-build.sh

```diff
@@ -19,15 +19,16 @@ set -o pipefail
 set -o xtrace
 
 readonly testinfra="$(dirname "${0}")/.."
-
-### job-env
-export KUBE_FASTBUILD=false
-export KUBE_GCS_RELEASE_SUFFIX=-cross  # to avoid colliding with 'kubernetes-build'
+readonly scenario='kubernetes_build.py'
+readonly scenario_args=(
+  --suffix=-cross
+  # --slow
```

Comment:
We will always set `KUBE_FASTBUILD=(true|false)` this is just a reminder for the future that we explicitly want to build all platforms (as opposed to others which may not care and just want the default, if we switch it to `store_false`)